data/colours.json

```diff
@@ -0,0 +1,108 @@
+{
+  "Main":{
+    "Text": [
+      {
+        "name": "$govuk-text-colour"
+      },
+      {
+        "name": "$govuk-secondary-text-colour"
+      }
+    ],
+    "Links": [
+      {
+        "name": "$govuk-link-colour"
+      },
+      {
+        "name": "$govuk-link-hover-colour"
+      },
+      {
+        "name": "$govuk-link-visited-colour"
+      }
+    ],
+    "Backgrounds": [
+      {
+        "name": "$govuk-border-colour"
+      },
+      {
+        "name": "$govuk-panel-colour"
+      },
+      {
+        "name": "$govuk-highlight-colour"
+      },
+      {
+        "name": "$govuk-page-colour"
+      }
+    ],
+    "Buttons": [
+      {
+        "name": "$govuk-button-colour"
+      },
+      {
+        "name": "$govuk-green",
+        "notes": "Use for button hover"
+      }
+    ],
+    "Focus": [
+      {
+        "name": "$govuk-focus-colour"
+      }
+    ],
+    "Status":[
+      {
+        "name": "$govuk-blue",
+        "notes": "Use for Alpha and Beta banners"
```

Comment:
Can we change to 'Use for phase banners' to be consistent with what we call them elsewhere in the Design System?

Reply:
I think this still needs doing, unless otherwise was decided.